CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributions
+
+For contributions please:
+1. Make sure your fork (fork it if it's the first time) is in sync with the parent repository.
+2. Make your changes in a topic branch in your fork.
+2. Make sure the changes pass all the tests & do not break anything.
+3. When ready create a pull request from your fork to the parent repository.
+
+# Local Setup
+
+## Initial Setup
+
+### Pre-requisites
+
+1. Install Rust: https://www.rust-lang.org/tools/install
+2. Install Docker: https://docs.docker.com/engine/install
```

Comment:
or any docker compose compatible solution like `podman`

Reply:
👍 addressed in https://github.com/liveask/liveask/pull/81/commits/350655e2b99ef04a032f600bcfc15a44fd0aef12